src/composables/incomingTransactions.ts

```diff
@@ -0,0 +1,72 @@
+import VueRouter from 'vue-router';
+import { roundAmountToPrecision } from '../popup/utils';
+import { i18n } from '../store/plugins/languages';
+import { IDefaultComposableOptions, ITransaction } from '../types';
+import { useAccounts } from './accounts';
+import { useNotifications } from './notifications';
+import { useGetter } from './vuex';
+import { useWebSocket } from './websocket';
+import { ROUTE_TX_DETAILS } from '../popup/router/routeNames';
+
+export interface IIncomingTransactionsOptions extends IDefaultComposableOptions {
+  router: VueRouter
+}
+
+export function useIncomingTransactions({ store, router }: IIncomingTransactionsOptions) {
+  const { watchTransactionsUpdates } = useWebSocket({ store });
+  const { addWalletNotification } = useNotifications({ store });
+  const { isLocalAccountAddress, getAccountNameByAddress } = useAccounts({ store });
+  const getTxAmountTotal = useGetter<any>('getTxAmountTotal');
+  const getTxSymbol = useGetter<any>('getTxSymbol');
```

Comment:
I think both of those could be typed

Reply:
No need for now. We are using this methods in many places and we didn't type them. But in the future... :)